test/index.test.js

```diff
@@ -1,14 +1,16 @@
 // @flow
 const test = require('tape-promise/tape')
-const fs = require('fs')
+const fs = require('fs-extra')
 const Zip = require('../bundle')
 
 test('index: create zip file', async (t) => {
   const zipfile = 'test/output.zip'
 
   t.pass('Reading directories "fixtures"')
 
-  fs.unlinkSync(zipfile)
+  if (fs.pathExistsSync(zipfile)) {
+    fs.unlinkSync(zipfile)
+  }
```

Comment:
Using `pathExists` before running an operation is an anti-pattern. Just run `unlinkSync`, then catch the error, if it's `ENOENT`, just ignore the error.

Reply:
See: https://github.com/ExodusMovement/zip-up/pull/4